src/main/java/online/book/store/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,26 @@
 package online.book.store.dto;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateBookRequestDto {
-    private String title;
+    @NotNull
+    @Size(min = 5, message = "author size must be min 5")
     private String author;
+    @NotNull
+    @Size(min = 5, message = "title size must be min 5")
+    private String title;
+    @NotNull
+    @Size(min = 10, max = 13)
```

Comment:
If I am right, we will be able to enter 11 or 12 characters, which is not correct